Documentation for Ekey Bionyx Integration (#37646)
home-assistant/home-assistant.github.io: source/_integrations/ekeybionyx.markdown

---
title: Ekey Bionyx
description: Instructions on how to integrate Ekey Bionyx devices into Home Assistant.
ha_category:
  - Event

> ha_release: 2025.7.0

Reviewer: ```suggestion
ha_release: 2025.10
```
To be checked when integration is merged.

Author: Changed it to 2025.10 for now

ha_iot_class: Local Push
ha_config_flow: true
ha_domain: ekeybionyx
ha_codeowners:
  - '@richardpolzer'
ha_platforms:
  - event
ha_integration_type: integration
---

> The `ekeybionyx` integration allows Home Assistant to receive events from your [ekey fingerprint reader](https://www.ekey.net).

Reviewer: ```suggestion
The **Ekey Bionyx** {% term integration %} allows Home Assistant to receive events from your [ekey fingerprint reader](https://www.ekey.net).
```

{% include integrations/config_flow.md %}

> ## Event Platform

Reviewer: ```suggestion
## Supported functionality

The **Ekey Bionyx** integration provides the following entities.

### Events
```
I think you can use the standard title and intro paragraph here 
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L88

The webhooks from the ekey device are represented in Home Assistant as events. You can use these events as triggers in automations.

> ## Current Limitations

Reviewer: ```suggestion
## Limitations
```

- Ekey allows only five webhooks per device
- You cannot link multiple Home Assistant instances to the same ekey system

> ## Installation Instructions

Reviewer: This section can be moved into a Prerequisites section above the config_flow.

Author: Moved it

To use the integration, the ekey device needs to be set up in **plus mode**, and the ekey Bionyx Third-Party API needs to be enabled in the app.

![Activate Ekey Bionyx Third Party API](/images/integrations/ekeybionyx/activate_third_party_api.png)

{% tip %}
Prefer using local IPs when setting up the integration instead of domain names because the events will be more responsive.
{% endtip %}

Once Home Assistant is set up, open the ekey app and assign events to the fingers you wish to use as triggers

> ## Uninstallation Instructions

Reviewer: ```suggestion
## Removing the integration
```

Based on the limited lifetime of the token, you have two options for cleanly resetting the connection to Home Assistant:

> - Delete the integration in Home Assistant, set it up again (triggering the deletion process), and confirm the deletion in the ekey app afterward.

Reviewer: For the steps on removing the integration from Home Assistant, we have a text bock you can use. See also http://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L200

which is rendered as follows:

<img width="347" height="165" alt="image" src="https://github.com/user-attachments/assets/1dd4243a-dc71-45dd-9edc-7870b5d15e03" />

Maybe you can integration this into your removal procedure.
{% include integrations/remove_device_service.md %}

Author: Since it is not the standard removal I don't know how to integrate that one I would need to extend it with custom steps.

---
title: Ekey Bionyx
description: Instructions on how to integrate Ekey Bionyx devices into Home Assistant.
ha_category:
  - Event
ha_release: "2025.10"
ha_iot_class: Local Push
ha_config_flow: true
ha_domain: ekeybionyx
ha_codeowners:
  - '@richardpolzer'
ha_platforms:
  - event
ha_integration_type: integration
---

The **Ekey Bionyx** {% term integration %} allows Home Assistant to receive events from your [ekey fingerprint reader](https://www.ekey.net).

> 

Reviewer: ```suggestion

## Prerequisites

```

> To use the integration, the ekey device needs to be set up in **plus mode**, and the ekey Bionyx Third-Party API needs to be enabled in the app.

Reviewer: ```suggestion
- Ekey Bionnyx app.
- The Ekey device needs to be set up in **plus mode** in the app.
- The Ekey Bionyx Third-Party API needs to be enabled in the app.
```

![Activate Ekey Bionyx Third Party API](/images/integrations/ekeybionyx/activate_third_party_api.png)

{% tip %}
Prefer using local IPs when setting up the integration instead of domain names because the events will be more responsive.
{% endtip %}

> Once Home Assistant is set up, open the ekey app and assign events to the fingers you wish to use as triggers

Reviewer: should this be moved below the config flow? Or what do you mean by "Once Home Assistant is set up? Once the integration is added to Home Assistant?

Author: Yes i meant once the integration is added there is another step required in the app to assign actions to the fingerprints that should trigger it

{% include integrations/config_flow.md %}

## Supported functionality

The **Ekey Bionyx** integration provides the following entities.

### Events

> The webhooks from the ekey device are represented in Home Assistant as events. You can use these events as triggers in automations.

Reviewer: There are different spellings of Ekey Bionyx. Which one is correct?

Author: Which ones are you reffering to? there is the Ekey device which is the hardware, and the Ekey Bionyx system which is the software plattform

Reviewer: The ones that are not capitalized. An when you say ekey system, do you mean Ekey Bionyx system?

Author: After reviewing their website it seems like it shouldn't be capitalized at all. https://www.ekey.net/en/the-scope-of-features/ How would you do at the beginning of sentences

Reviewer: I would try and work around it. For example, by adding an article.

## Limitations

- Ekey allows only five webhooks per device

> - You cannot link multiple Home Assistant instances to the same ekey system

Reviewer: ```suggestion
- Ekey allows only five webhooks per device.
- You cannot link multiple Home Assistant instances to the same ekey system.
```

---

> title: ekey bionyx

Author: Here I'm not certain if I should leave it capitalized. However it doesn't change the appearance in the list

Reviewer: I would leave it non-capitalized in that case, like you did.

description: Instructions on how to integrate ekey bionyx devices into Home Assistant.
ha_category:
  - Event
ha_release: "2025.10"
ha_iot_class: Local Push
ha_config_flow: true
ha_domain: ekeybionyx
ha_codeowners:
  - '@richardpolzer'
ha_platforms:
  - event
ha_integration_type: integration
---

The **ekey bionyx** {% term integration %} allows Home Assistant to receive events from your [ekey fingerprint reader](https://www.ekey.net).

## Prerequisites

> - ekey bionyx app.

Author: this capitalization looks kinda off now.

> - ekey bionyx app.

Reviewer: ```suggestion
- The **ekey bionyx** app.
```
In these cases, I usually try to work around it by adding an article.

- The ekey device needs to be set up in **plus mode** in the app.
- The ekey bionyx Third-Party API needs to be enabled in the app.

![Activate ekey bionyx Third Party API](/images/integrations/ekeybionyx/activate_third_party_api.png)

{% include integrations/config_flow.md %}

{% tip %}
Prefer using local IPs when setting up the {% term integration %} instead of domain names because the events will be more responsive.
{% endtip %}

Once the {% term integration %} is added, open the ekey app and assign events to the fingers you wish to use as triggers.

## Supported functionality

The **ekey bionyx** {% term integration %} provides the following entities.

### Events

The webhooks from the ekey device are represented in Home Assistant as events. You can use these events as triggers in automations.

## Limitations

> - Ekey allows only five webhooks per device.

Author: since I'm refering to the company and its the first word of the sentence I left it capitalized

> - Ekey allows only five webhooks per device.

Reviewer: ```suggestion
- The ekey integration allows only five webhooks per device.
```

The **Ekey Bionyx** {% term integration %} allows Home Assistant to receive events from your [ekey fingerprint reader](https://www.ekey.net).

## Prerequisites

- Ekey Bionyx app.
- The Ekey device needs to be set up in **plus mode** in the app.
- The Ekey Bionyx Third-Party API needs to be enabled in the app.

![Activate Ekey Bionyx Third Party API](/images/integrations/ekeybionyx/activate_third_party_api.png)

Once the {% term integration %} is added, open the ekey app and assign events to the fingers you wish to use as triggers

> 

Reviewer: ```suggestion
```

{% include integrations/config_flow.md %}

{% tip %}
Prefer using local IPs when setting up the integration instead of domain names because the events will be more responsive.
{% endtip %}

> 

Reviewer: ```suggestion

```suggestion

Once the {% term integration %} is added, open the **ekey bionyx** app and assign events to the fingers you wish to use as triggers.

```
Does this belong after the config flow? It's one after the integration has been added to Home Assistant. 
<img width="885" height="600" alt="image" src="https://github.com/user-attachments/assets/cf6e5dc3-c2d0-4a0e-886b-75d55ba53be9" />

```

Author: Put it underneath the tip
